### DISCUSSION TITLE
No ISOs found

### DISCUSSION THREAD

**Discussion by mBrane** [ventoy/Ventoy#2394] [Q&A]
I have an external USB drive that has Ventoy 1.0.91 installed (256 GB, exFAT, GPT).  I see the large exFAT partition and the small/32 MB FAT partition with the Ventoy files.

I had a couple of ISOs loaded into the exFAT partition and was able to boot to Ventoy and launch those ISOs just fine (even in SecureBoot mode, after loading Ventoy EFI key into TPM).

I booted to Windows, copied some more ISOs to the exFAT partition, and now when I boot Ventoy, it sees *none* of the ISOs!  Bizarre.  Everything used to work and still looks healthy in Ventoy2Disk.  The only thing that changed is adding more ISOs.  Any ideas out there?  Thanks in advance!

**Comment by steve6375** [comment]
Bad or fake usb drive.
Cheap fake usb drives contain less memory than advertised .  When you write enough sectors, they start to overwrite the bottom of existing memory again, thus filesystem is corrupted

**Reply by mBrane** [reply]
Thank you, but no, this is a known-good Corsair Force GT SSD that I've been using for quite some time.

Plus, Windows can still access the drive and partitions just fine and calc correct CRC hashes for ISOs.  This disk is solid:

![image](https://github.com/ventoy/Ventoy/assets/12666288/73dadf70-13d6-4fbe-9000-4d30d496cf15)
![image](https://github.com/ventoy/Ventoy/assets/12666288/1870483f-aee4-4a06-92ee-f3fca16a9e67)

**Reply by steve6375** [reply]
Does the ventoy.json file exist? Try deleting it?
If you press c in Ventoy and do an ls command, can you list ISO files?
Same in uefi and legacy boot?
Try on different system ? Try different usb ports?

**Reply by e-t-l** [reply]
I'm having i think a similar issue to the OP. When you say to press `c` in Ventoy then `ls`, where should the iso files be listed? I similarly just have a list of disks like `(proc) (hd0) (hd1)` etc...

**Comment by mBrane** [comment]
_Does the ventoy.json file exist? Try deleting it?_ - no

_If you press c in Ventoy and do an ls command, can you list ISO files?_
![IMG_4897](https://github.com/ventoy/Ventoy/assets/12666288/30ee636c-6d06-41ef-bbb6-3f1f3d5f1859)

_Same in uefi and legacy boot?_ - yes

_Try different usb ports?_ - same USB-C port that worked earlier.  Windows has no problem mounting disk on same port throughout.

_Try on different system ?_ - trying now...

**Comment by steve6375** [comment]
ls (hd0,gpt1)/
will list files in root of partition 1.
Change path to list ISOs (wherever you have put them).
A USB Port may work OK under Windows (which has drivers), but may not work under grub2 (uses different drivers, etc.)

**Comment by mBrane** [comment]
I appreciate your help, Steve.  I've found the problem.
* the drive worked the same on a Win11 Pro machine, BTW.

What I found is that my Windows 11 Home laptop is doing an unprompted Bitlocker encrypt of my USB drive!  WTF?!  That's straight *ransomware* behavior!  This is *criminal* default behavior for Windows!  Clearly not Ventoy's fault!  Googling this reveals old info that - if I had an MDM tool like Intune - this can be disabled.  Of course, there is no GPO snap-in for Win11 Home.  That leaves one single global toggle to disable Bitlocker for entire machine.  Grrr...

I dropped to PowerShell and disabled Bitlocker on the Ventoy volume.  Rebooted.  Ventoy working once again!  Windows still hasn't tried to re-encrypt volume, so maybe it remembers when Bitlocker has been disabled for a particular drive?  Again, I've only found PowerShell effective for this, which doesn't make sense for most people running Win11 Home, but I digress...

Now to figure out how to boot most of the ISO's that I have that complain, "Maybe this image does not support X64 UEFI."

Again, thank you for your speedy engagement, Steve!
